assets/elasticsearch/Dockerfile

```
@@ -0,0 +1,24 @@
+FROM ghcr.io/y-scope/clp/clp-core-dependencies-x86-ubuntu-jammy:main
+
+RUN apt-get update \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)